polaris/ocean/tests/single_column/forward.py

```diff
@@ -72,6 +72,10 @@ def __init__(self, test_case, name='forward', subdir=None,
         self.add_yaml_file('polaris.ocean.tests.single_column',
                            'forward.yaml')
 
+        if ideal_age:
+            self.add_yaml_file('polaris.ocean.tests.single_column.ideal_age',
+                               'forward.yaml')
```

Comment:
What you did here is fine.  The main thing is that it might become tricky as we have more and more single-column test cases.  If we can pull out pieces specific to test cases, it would keep things simpler.

In this case, we can do this inside of the test case's `__init__()`.  I'll suggest changes below.